Marlin/example_configurations/makibox/Configuration_adv.h

```diff
@@ -220,13 +220,17 @@
 #define EXTRUDER_AUTO_FAN_TEMPERATURE 50
 #define EXTRUDER_AUTO_FAN_SPEED   255  // == full speed
 
-// Define a pin to turn case light on/off
-//#define CASE_LIGHT_PIN 4
-#if PIN_EXISTS(CASE_LIGHT)
-  #define INVERT_CASE_LIGHT false   // Set to true if HIGH is the OFF state (active low)
-  //#define CASE_LIGHT_DEFAULT_ON   // Uncomment to set default state to on
-  //#define MENU_ITEM_CASE_LIGHT    // Uncomment to have a Case Light On / Off entry in main menu
-#endif
+// M355 Case Light command 
+//   Set case light brightness/on/off
+
```

Comment:
No blank line between the option's description and the option.
However, do insert a blank line after `#define CASE_LIGHT_ENABLE`.